gateway/rpc_storage_handler.go

```diff
@@ -1116,6 +1120,14 @@ func (r *RPCStorageHandler) ProcessKeySpaceChanges(keys []string, orgId string)
 		}
 	}
 
+	for _, apiID := range apiIDsToDeleteCache {
+		if r.Gw.invalidateAPICache(apiID) {
+			log.WithField("apiID", apiID).Info("cache invalidated")
+			continue
+		}
+
+		log.WithField("apiID", apiID).Info("cache invalidation failed")
```

Comment:
What value does the logging here add, does it need to exist?

Reply:
currently, there is not fallback strategy for rpc processing in case redis is down on edge cluster. logs would be needed in such cases if cache isn't invalidated

Reply:
Ok, let's asume it's useful to log the unhappy flow - raise it to a `error` from `info`, and consider removing the log on the happy flow a few lines higher.